app/api/v1/users.py

```
@@ -0,0 +1,69 @@
+from app.core.memory_service import MemoryService
+from app.dependencies import get_memory_service
+import logging
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.post("", response_model=UserResponse)
+async def add_user(
+    user_id: str,
+    memory_service: MemoryService = Depends(get_memory_service),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)